src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -9,7 +9,8 @@ import {
   View
 } from 'react-native';
 import {colors} from '../../../utils/colors';
-import {fonts} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
+import {CircleGradient} from '../../../components/Karma/CircleGradient';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include the `CircleGradient` component from the Karma module. This is a new addition and seems to be correctly imported.
```diff
- import {fonts} from '../../../utils/fonts';
+ import {fonts, normalize} from '../../../utils/fonts';
+ import {CircleGradient} from '../../../components/Karma/CircleGradient';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->